master/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -134,6 +135,9 @@ spec:
             # Set Felix logging to "info"
             - name: FELIX_LOGSEVERITYSCREEN
               value: "info"
+            # Set MTU for tunnel device
```

Comment:
We do have the flag `FELIX_IPINIPENABLED` as true in the manifest, should I put `Set MTU for tunnel device used because IP-in-IP is enabled`?